src/pykeen/nn/representation.py

```diff
@@ -124,6 +124,11 @@ class Representation(nn.Module, ExtraReprMixin, ABC):
     #: dropout
     dropout: nn.Dropout | None
 
+    # TODO: cyclic import
+    # @update_docstring_with_resolver_keys(ResolverKey("normalizer", resolver="pykeen.nn.normalizer_resolver"))
```

Comment:
will look into this

Reply:
Work-around coming in https://github.com/cthoyt/class-resolver/pull/71

Reply:
that didn't solve it :/

Reply:
https://github.com/cthoyt/class-resolver/pull/72 might have done it. locally it looks like it worked